install/linux/usr/share/odemis/hwtest/sr193-andor3.odm.yaml

```diff
@@ -0,0 +1,128 @@
+# Configuration to test just a spectrometer SR193i + Andor SDK3 camera
+SPARC-Spec: {
+    class: Microscope,
+    role: sparc,
+    children: ["SEM E-beam", "SEM Detector", "Optical Path Properties",
+               "External Spectrograph", "Spec CCD Flipper", "Spectrograph focus",
+               "Spectral Camera", "Spectrometer"],
+}
+
+# This is a more realistic SEM simulator (image-wise, but less realistic HW-wise)
+"SEM Scan Interface": {
+    class: simsem.SimSEM,
+    role: null,
+    init: {
+           drift_period: 5, # s  (disable to get a the whole resolution of the image)
+           image: "sparc-etd-gold-balls.h5",
+    },
+    # more detectors can be added, if necessary
+    children: {scanner: "SEM E-beam",
+               detector0: "SEM Detector"
+    }
+}
+
+"SEM E-beam": {
+    # Internal child of SimSEM, so no class
+    role: e-beam,
+    init: {},
+    properties: {
+        dwellTime: 10.e-6, # s
+    },
+    affects: ["SEM Detector", "Spectrometer", "Spectral Camera"] # affects the CCD in case of cathodoluminescence
+}
+
+"SEM Detector": { # aka ETD
+    # Internal child of SimSEM, so no class
+    role: se-detector,
+    init: {},
+}
+
+
+# Spectrometer = Andor iVac + Shamrock SR193 (connected via USB)
+"Spectrometer": {
+    class: spectrometer.CompositedSpectrometer,
+    role: spectrometer,
+    dependencies: {detector: "Spectral Camera", spectrograph: "External Spectrograph"},
+    init: {
+        transp: [1, 2], # only applied to the spectrometer data (not raw CCD)
+    },
+    properties: {
+        #binning: [4, 1080],
+    },
+}
+
+"Spectral Camera": {
+    class: andorcam3.AndorCam3,
+    role: ccd,
+    init: {
+           device: 0,  # if a real camera is plugged in, it's usually 0
+           bitflow_install_dirs: "/usr/share/bitflow/", # Put the path where Bitflow data is installed
+           #max_bin: [640, 16], # (px), to limit the maximum binning on X, Y
```

Comment:
Is it correct, that it is a comment? As you introduced the max_bin in the first commit here. Btw, is there a test case for testing the correct bg level subtraction?

Reply:
Yes, because for the testing in general, we don't want to use this limited binning. This limited binning is only used on some cameras, for certain systems.

Reply:
I've created a test case for the software binning (and baseline level subtraction)

Reply:
> Yes, because for the testing in general, we don't want to use this limited binning. This limited binning is only used on some cameras, for certain systems.

So maybe good to mention that here, so the next reader knows why it is commented code?